packages/fxa-graphql-api/src/gql/dto/input/reject-unblock-code.ts

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Field, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class RejectUnblockCodeInput {
+  @Field({
+    description: 'A unique identifier for the client performing the mutation.',
+    nullable: true,
+  })
+  public clientMutationId?: string;
+
+  @Field()
+  public uid!: string;
```

Comment:
Not really for this PR, but auth-server does some validation on uid and codes before using them. Is the equalivent here to specify types?

Reply:
huh that's a good point.  We don't seem to do any additional validation beyond basic types in gql-api.